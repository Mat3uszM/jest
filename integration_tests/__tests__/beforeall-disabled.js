// @flow

'use strict';

const runJest = require('../runJest');
const {extractSummary} = require('../utils');

it('should not run beforeall for disabled tests', () => {
  const result = runJest('beforeall-disabled', ['-t abc']);

  expect(result.stdout).toMatchSnapshot();
  expect(extractSummary(result.stderr).rest).toMatchSnapshot();
});
