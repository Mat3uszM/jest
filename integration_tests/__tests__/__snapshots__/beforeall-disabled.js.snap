// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not run beforeall for disabled tests 1`] = `""`;

exports[`should not run beforeall for disabled tests 2`] = `
"PASS ./skip.test.js
  abc
    ✓ runs abc
  def
    ○ skipped 1 test

  ● def › does not run def

    Failed: Ran beforeAll for def
      
      at ../../packages/jest-jasmine2/build/expectation_result_factory.js:49:427
      at skip.test.js:6:19

"
`;
